File: sdks/python/src/bigrag/resources/admin/_shared.py
from __future__ import annotations


def _pagination(*, limit: int | None, offset: int | None) -> dict[str, str]:
    params: dict[str, str] = {}
    if limit is not None:
        params["limit"] = str(limit)
    if offset is not None:
        params["offset"] = str(offset)
    return params

File: sdks/typescript/src/resources/admin/_shared.ts
export function pagination(options: { limit?: number; offset?: number }): Record<string, string> {
  const params: Record<string, string> = {};
  if (options.limit !== undefined) params.limit = String(options.limit);
  if (options.offset !== undefined) params.offset = String(options.offset);
  return params;
}
